consoles/VNC.pm

```diff
@@ -755,15 +757,15 @@ sub map_and_send_key {
     if (!defined $down_flag || $down_flag == 1) {
         for my $key (@events) {
             $self->send_key_event_down($key);
+            usleep($press_release_delay_us);
         }
     }
-    usleep(2_000);
```

Comment:
danger mouse! you only sleep between combined keys and no longer between down and up of single keys.

And take the chance to raise the default delay - for all we know a little slower typing makes testing more stable at an acceptable price.

Reply:
So, you want to:
* send X keys down at a time
* wait a bit
* send X keys up at a time
?
Not sure it will really enhance things.

If we wait between each key, I think it should be more effective. But I cannot get why it could break. On my machine I can press `alt` wait 10 seconds, press `v` wait 10 seconds, release `v`, wait 10 seconds, release `alt` and all works as ecpected.

Reply:
No, I want to wait after every action -and one extra time before the key up actions.

Reply:
It is not a big deal to add this extra time, but not sure why it would be needed.

Reply:
Added it back now.